document/document_test.go

```diff
@@ -510,6 +536,50 @@ func TestNewFromStruct(t *testing.T) {
 		require.NoError(t, err)
 		require.EqualValues(t, 2, v.V.(int64))
 	})
+
+	t.Run("Embedded struct", func(t *testing.T) {
+		db, err := genji.Open(":memory:")
+		require.NoError(t, err)
+		defer db.Close()
+
+		ctx := context.Background()
+		err = db.Exec(ctx, "CREATE TABLE test")
+		require.NoError(t, err)
+
+		type baz struct {
+			I int `genji:"II"`
+			A string
+		}
+
+		type bar struct {
+			Baz  baz
+			Id   int    `genji:"-"`
+			Name string `genji:"tag-name"`
+		}
+
+		type foo struct {
+			*bar
+			B   string `genji:"b-b"`
+			Bar *bar
+			i   int // ignored
+			A   []byte
+		}
+
+		bb := baz{I: 100, A: "ptr"}
+		e := bar{Id: 1, Name: "Foo", Baz: bb}
+		f := foo{bar: &e, Bar: &e, B: "BString", A: []byte("foo"), i: 300}
+		doc, err := document.NewFromStruct(&f)
+		require.NoError(t, err)
+		v, err := doc.GetByField("id")
+		require.NoError(t, err)
+		require.EqualValues(t, 1, v.V.(int64))
```

Comment:
`Id` has the `genji:"-"` tag, it should be ignored right?

Reply:
Yes I misunderstand, the field should be ignored.